src/index.js

```diff
@@ -47,6 +49,8 @@ exports.RecordCreator = Interface.RecordCreator;
 exports.RecordRemover = Interface.RecordRemover;
 exports.RecordsRemover = Interface.RecordsRemover;
 exports.RecordSerializer = Interface.RecordSerializer;
+exports.BaseOperatorDateParser = Interface.BaseOperatorDateParser;
```

Comment:
exporting classes (like everywhere in this repo) in an anti-pattern.
we should export API and do not export class definition which is not a sharable unit.